docs/configuration.rst

```diff
@@ -307,3 +307,36 @@ These configurations control if the default pipelines should be enabled or not
     - false
     - bool
     - Whether Pitaya should enable the default struct validator for handler arguments
+
+Groups
+=================
+
+These configurations are only used if groups with ETCD implementation are used.
+
+.. list-table::
+  :widths: 15 10 10 50
+  :header-rows: 1
+  :stub-columns: 1
+
+  * - Configuration
+    - Default value
+    - Type
+    - Description
+  * - pitaya.groups.etcd.endpoints
+    - localhost:2379
+    - string
+    - Comma separated list of etcd endpoints to be used by the binding storage module, should be the same as the service discovery etcd
```

Comment:
@rodrigolck @Henrod if these must be the same as the service discovery etc configs why are we using another variable?

Reply:
My mistake, wrote the wrong config

Reply:
It doesn't need to be the same etcd. Using the same config would not allow us to use different etcds for each task. 

@rodrigolck that description is wrong, it is not used for binding storage nor service discovery